src/Caliburn.Micro.Avalonia/Platforms/netcore-avalonia/BootstrapperBase.cs

```diff
@@ -102,13 +101,12 @@ protected virtual void StartRuntime()
         /// </summary>
         protected virtual void PrepareApplication()
         {
-            //TODO: (Avalonia) Implement application events when Avalonia supports them
-            
-            //Application.Startup += OnStartup;
-
-            //Aplication.DispatcherUnhandledException += OnUnhandledException;
+            if (Application.Current.ApplicationLifetime is IControlledApplicationLifetime controlledApplicationLifetime)
```

Comment:
This is potentially confusing.
For iOS, Android and Blazor apps this is ISingleViewApplicationLifetime and it does not have the events. So the user could override the methods and think they will be called but they never will.

What do we think about this?